user-metrics/metrics-service-datadog/metrics-service.yml.template

```diff
@@ -25,10 +25,3 @@ spec:
         limits:
           memory: 200M
           cpu: 200m
-  endpoints:
```

Comment:
why do we delete these endpoints?

Reply:
This is a datadog example and these endpoint will not have any relevance hence removed them